ql/src/java/org/apache/hadoop/hive/ql/plan/PlanUtils.java

```diff
@@ -906,6 +907,15 @@ private static void configureJobPropertiesForStorageHandler(boolean input,
                     "using configureTableJobProperties",e);
                 storageHandler.configureTableJobProperties(tableDesc, jobProperties);
             }
+
+            try{
+              storageHandler.configureInputJobCredentials(
+                tableDesc,
+                jobSecrets);
+            } catch(AbstractMethodError e) {
+              // ignore
+              LOG.info("configureInputJobSecrets not found");
```

Comment:
Looked at it some more. It's useful to know that the StorageHandler is of an older version and that the method wasn't invoked. We're doing the same thing for other methods.